news/three-months-at-the-rust-foundation.md

```diff
@@ -0,0 +1,83 @@
+
+## Support for the crates.io support
+
+Until recently, [Carol (Nichols || Goulding)](https://github.com/carols10cents) had primarily managed the [help@crates.io](mailto:help@crates.io) support email address on her own. Since I was now getting paid to work on crates.io, I offered to support her with this too.
+
+My first task was updating the (private) "crates.io operations guide", which also contains information on how to handle common support requests, like people requesting crate transfers or crate deletions due to privacy-related concerns.
+
+While the support volume is fortunately still limited to only a couple of requests per day, this number quickly adds up when you can only handle these requests once in a while. I've committed myself to checking for new support emails each morning now, and with Carol covering the US timezones, we should be able to respond to your requests in a somewhat timely manner now.
+
+## Dependencies
+
+We have been using [Renovate](https://github.com/renovatebot/renovate) on the crates.io repository for quite a while already, and it is a great way to keep all of our dependencies up-to-date. Still, sometimes a dependency update can't just be merged because it is breaking our continuous integration checks in some way. When I started my job in April we had quite a few blocked dependency upgrades due to such issues. One week later, I managed to upgrade us to the latest version of everything again.
+
+During this time, I also discovered [cargo-deny](https://github.com/EmbarkStudios/cargo-deny), and later incorporated it into our CI checks. This tool assists in identifying problematic dependencies in our dependency tree, including those with incompatible licenses or active security advisories. Upon its implementation, we promptly addressed five issues it had identified.
+
+## Closing thoughts
+
+While it often feels like each week yields only modest progress, my [work log](https://blog.pragmaticengineer.com/work-log-template-for-software-engineers/) has already grown to four pages and includes a lot more bullet points than I could fit in this blog post. I'm also pretty sure that this new job significantly increased my [imposter syndrome](https://blog.rust-lang.org/inside-rust/2022/04/19/imposter-syndrome.html), but since people keep telling me that they are happy with the work that I'm doing, I guess I'm doing alright…
+
+Finally, I would like to thank the [OpenSSF's Alpha-Omega Initiative](https://openssf.org/community/alpha-omega/) and the [JFrog](https://jfrog.com/blog/jfrog-joins-rust-foundation-as-platinum-member/) for funding the Rust Foundation security initiative, which enabled me to get this job and work on a lot of the things above.
```

Comment:
Small note, JFrog are providing in-kind support rather than money, so it may be best to say "supporting" instead of "funding".